py/server/deephaven/experimental/s3.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+#
+"""
+    This module is useful for reading and writing files stored in S3. Importing this module requires the S3 specific
+    extensions to be included in the class path.
+"""
+
+import jpy
+
+_JDuration = jpy.get_type("java.time.Duration")
+_JS3Instructions = jpy.get_type("io.deephaven.extensions.s3.S3Instructions")
+
+def _build_s3_instructions(
+        aws_region_name: str,
+        max_concurrent_requests: int = None,
+        read_ahead_count: int = None,
+        fragment_size: int = None,
+        max_cache_size: int = None,
+        connection_timeout: _JDuration = None,
+        read_timeout: _JDuration = None,
+):
+    """
+    Build specialized instructions for accessing files stored in AWS S3.
+
+    Args:
+        aws_region_name (str): the AWS region name for reading parquet files stored in AWS S3, by default None
+        max_concurrent_requests (int): the maximum number of concurrent requests for reading parquet files stored in S3,
+            by default 50.
+        read_ahead_count (int): the number of fragments to send asynchronous read requests for while reading the current
+            fragment, defaults to 1.
```

Comment:
Does 1 mean "the current fragment plus the next" or does it mean "the current fragment"?

Reply:
It means "the current fragment plus the next". I have tried to explain this in the comment.
Let me know if it needs more clarification.